src/app/vault/share.component.html

```diff
@@ -1,10 +1,10 @@
-<div class="modal fade" tabindex="-1" role="dialog" aria-modal="true" aria-labelledby="shareTitle">
+<div class="modal fade" tabindex="-1" role="dialog" aria-modal="true" aria-labelledby="moveSelectedToOrgTitle">
     <div class="modal-dialog" role="document">
         <form class="modal-content" #form (ngSubmit)="submit()" [appApiAction]="formPromise">
             <div class="modal-body">
                 <div class="box">
-                    <div class="box-header" id="shareTitle">
-                        {{'share' | i18n}}
+                    <div class="box-header" id="moveSelectedToOrgTitle">
```

Comment:
This is the `id` you used in the bulk share component on `web`, but this is not a bulk action. I think it should just be `moveToOrgTitle`.